modules/ocf_irc/manifests/nodejs/apt.pp

```diff
@@ -0,0 +1,18 @@
+class ocf_irc::nodejs::apt {
+  package {
+    'apt-transport-https':;
+  }
+
+  # Install a newer version of Node.js for the Slack bridge and web IRC app
+  apt::key { 'nodejs':
+    id     => '9FD3B784BC1C6FC31A8A0A1C1655A0AB68576280',
+    source => 'https://deb.nodesource.com/gpgkey/nodesource.gpg.key';
+  }
```

Comment:
We'd only want to backport for flood though, not for all servers, so doesn't this mean that backporting would have to be fixed first?